parcels/fieldset.py

```diff
@@ -636,6 +636,8 @@ def from_pop(cls, filenames, variables, dimensions, indices=None, mesh='spherica
                       W nodes are at the centre of the horizontal interfaces.
                       They are interpolated linearly (as a function of z) in the cell.
                       T node is at the cell centre, and constant per cell.
+                      Note that Parcels assumes that the length of the depth dimension (at the W-points)
+                      is one larger than the size of the velocity and tracer fields in the depth dimension.
```

Comment:
Is this how pop-data is usually provided?

Reply:
Apparently, but it would be good to confirm that with other users

Reply:
For me, as a user of POP, there is two depths provided: 
1. The w_dep, which is the depth of the W-points and has a size which is one larger than velocity and tracer fields in the depth dimension.
2. The depth_t, which is the depth at the T-points and has the same size as the velocity and tracer fields in the depth dimension.